modules/client-configuration/pages/supported-features-sles.adoc

```diff
@@ -22,149 +22,205 @@ The icons in this table indicate:
 * {question} the feature is under consideration, and may or may not be made available at a later date
 
 
-[cols="1,1,1,1", options="header"]
+[cols="1,1,1,1,1,1", options="header"]
 .Supported Features on {suse} and {opensuse} Operating Systems
 |===
 
 | Feature
 | {sle}{nbsp}12
 | {sle}{nbsp}15
-| {opensuse}{nbsp}15
+| {sle}{nbsp}16
+| {opensuse}{nbsp}Leap 15
+| {opensuse}{nbsp}Leap 16
 
 | Client
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System packages
 | {suse}
 | {suse}
+| {suse}
+| {opensuse} Community
 | {opensuse} Community
 
 | Registration
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Install packages
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Apply patches
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Remote commands
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System package states
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System custom states
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Group custom states
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Organization custom states
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System set manager (SSM)
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Product migration
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Basic Virtual Guest Management {star}
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Advanced Virtual Guest Management {star}
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Virtual Guest Installation ({ay}), as Host OS
 | {cross}
 | {cross}
 | {cross}
+| {check}
+| {check}
 
 | Virtual Guest Installation (image template), as Host OS
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Virtual Guest Management
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System deployment (PXE/{ay})
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | System redeployment ({ay})
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Contact methods
 | {cross}
 | {cross}
+| {cross}
+| {check} ZeroMQ, Salt-SSH
 | {check} ZeroMQ, Salt-SSH
 
 | Works with {productname} Proxy
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Action chains
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Staging (pre-download of packages)
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | Duplicate package reporting
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | CVE auditing
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
 
 | SCAP auditing
 | {check}
 | {check}
 | {check}
+| {check}
+| {check}
```

Comment:
We don't have scap content for it available yet either so change them to question mark